docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -16,46 +16,55 @@ This means less manual work for workflow developers, and less waiting for conten
 
 This requires the addition of a ``/.dockstore.yml`` file to your repository on GitHub.
 This file contains workflow information such as workflow path, test parameter paths, etc. that Dockstore will use to setup
-the corresponding workflow on Dockstore. It's important to note, that you will need a ``/.dockstore.yml`` file on each branch of your GitHub
+the corresponding workflow on Dockstore. Branches that do not have a ``/.dockstore.yml`` file on each branch of your GitHub
```

Comment:
?

Reply:
Forgot to save the edit to the second half the sentence. Intent was:
`Branches that do not have a ``/.dockstore.yml`` file will not be synchronized.`

Reason for changing was that previous wording implied that if you wanted _any_ add'l branches to be sync'd, you might need _all_ of them to have a yml, ie, if you wanted /develop/ and /master/ on GATK you would need to write hundreds of yml files including stale branches you don't care about.